usaspending_api/references/reference_helpers.py

```diff
@@ -5,6 +5,10 @@
 from usaspending_api.accounts.models import FederalAccount, TreasuryAppropriationAccount
 
 
+def remove_federal_accounts(tas_tuple=None):
```

Comment:
Could we get a docstring on this?  Just `remove_federal_accounts` doesn't answer many questions (remove which ones?  Remove them why?)

... granted, a single line of code is not hard to read, but still.